adapter/statsd/config/BUILD

```diff
@@ -0,0 +1,19 @@
+load("@org_pubref_rules_protobuf//go:rules.bzl", "go_proto_library")
```

Comment:
please change everything to gogoprotobuf.

so: gogo:rules.bzl, gogo_proto_library.